rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp

```diff
@@ -841,29 +847,35 @@ void RobotLink::createMass(const urdf::LinkConstSharedPtr & link)
 void RobotLink::createInertia(const urdf::LinkConstSharedPtr & link)
 {
   if (link->inertial) {
-    // display a box sized as if it were a box of uniform density
-    // with the same inertia as the link
-    // Ixx = mass/12 (ly^2 + lz^2)
-    // Iyy = mass/12 (lx^2 + lz^2)
-    // Izz = mass/12 (lx^2 + ly^2)
-    urdf::Pose pose = link->inertial->origin;
-    Ogre::Vector3 translate(pose.position.x, pose.position.y, pose.position.z);
-    Ogre::Quaternion rotate(pose.rotation.w, pose.rotation.x, pose.rotation.y, pose.rotation.z);
+    const ignition::math::Vector3d Ixxyyzz(
```

Comment:
style nitpick: variables should be lowercase and snake_case ([style guide](https://google.github.io/styleguide/cppguide.html#Variable_Names)).